### DISCUSSION TITLE
Issue with MetaMask Transaction on Sepolia Testnet: High ETH Requirement

### DISCUSSION THREAD

**Discussion by Vishalsb11** [Cyfrin/foundry-full-course-cu#3134] [Q&A]
I'm currently encountering an issue with MetaMask while trying to get started with transactions on the Sepolia testnet. MetaMask is requesting 1 ETH to facilitate a transaction where I only need 0.5 Sepolia ETH. This seems unusually high for a testnet transaction, and it's hindering my progress with my first transaction.

Has anyone else experienced this issue? If so, how did you resolve it? Alternatively, are there any other solutions or workarounds that might help me bypass or reduce this requirement?

Any guidance would be greatly appreciated as I navigate this for the first time.

Thank you!

![Cyfrin_1ETH_Issue](https://github.com/user-attachments/assets/332a5058-9d8b-4467-bf7d-3d5e1553877a)

**Comment by EngrPips** [comment]
Hello @Vishalsb11, getting sepolia eth has been tough lately. Please try this faucet https://cloud.google.com/application/web3/faucet/ethereum/sepolia

**Reply by Vishalsb11** [reply]
Thank you it worked

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.

**Comment by H-code9** [comment]
@Vishalsb11 kindy implement the solution above, Best regards.

**Reply by EngrPips** [reply]
Great heads-up
